src/Sulu/Bundle/SecurityBundle/Resources/config/routing.yml

```diff
@@ -0,0 +1,25 @@
+sulu_security.reset_password.email:
+    path: reset/email
+    defaults:
+        _controller: SuluSecurityBundle:Resetting:sendEmail
+
+sulu_security.reset_password.email.resend:
+    path: reset/email/resend
+    defaults:
+        _controller: SuluSecurityBundle:Resetting:sendEmail
+        generateNewKey: false
+
+sulu_security.reset_password.reset:
+    path: reset
+    defaults:
+        _controller: SuluSecurityBundle:Resetting:reset
+
+sulu_security.contexts:
```

Comment:
is there a reason why these routes are not imported in the `routing_api.yml` file?